pkg/process/manager.go

```diff
@@ -296,6 +296,22 @@ func (m *Manager) Reconnect(ctx context.Context) error {
 		info, err := d.CheckStatus()
 		if err != nil {
 			log.L.WithField("daemon", d.ID).Warnf("failed to check daemon status: %v", err)
+
+			if d.ID == daemon.SharedNydusDaemonID {
+				// The only season that nydusd can't be connected is it's not running.
+				// Moreover, snapshotter is restarting. So no nydusd states can be returned to each nydusd.
+				// Nyudsd can't do failover any more.
+				// We can safely try to umount its mountpoint to avoid nydusd pausing in INIT state.
+				log.L.Infof("Nydusd died somehow. Clean up its vestige.")
```

Comment:
`Nydusd in shared mode died somehow, clean up its vestige.`, and maybe use warning level?